src/badge/withBadge.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import Badge from './Badge';
+
+const withBadge = (value, options = {}) => WrappedComponent => {
+  class WithBadge extends React.Component {
+    render() {
+      const {
+        left = 0,
+        bottom = 0,
+        top = -5,
+        right = 0,
+        hidden = !value,
```

Comment:
Also we need to change this logic here.

The badge also has a mini badge mode that shows when no value is given to the Badge. Currently, right now this will always be hidden if the user is trying to use the mini badge.

```jsx
// Badge will never show since `hidden = !value` will be true
const StatusAvatar = withBadge(undefined)(Avatar);
```

Think we may need to just set hidden to false by default instead of checking for the value. @xavier-villelegier

Reply:
Sounds logical.

Reply:
Yea let's do that

Reply:
Sounds good to me, go for it !

Reply:
```suggestion
        hidden = false,
```